src/commands/apps-sdkconfig.spec.ts

```diff
@@ -0,0 +1,188 @@
+import { expect } from "chai";
+import * as sinon from "sinon";
+import * as fs from "fs-extra";
+import { Command } from "../command";
+import * as apps from "../management/apps";
+import { AppPlatform } from "../management/apps";
+import * as projectUtils from "../projectUtils";
+import * as projects from "../management/projects";
+import { FirebaseError } from "../error";
+import * as prompt from "../prompt";
+import { command } from "./apps-sdkconfig";
+import * as auth from "../requireAuth";
+
+import { logger } from "../logger";
+
+describe("apps:sdkconfig", () => {
+  let sandbox: sinon.SinonSandbox;
+  let needProjectIdStub: sinon.SinonStub;
+  let getAppConfigStub: sinon.SinonStub;
+  let getAppConfigFileStub: sinon.SinonStub;
+  let listFirebaseAppsStub: sinon.SinonStub;
+  let getOrPromptProjectStub: sinon.SinonStub;
+  let selectStub: sinon.SinonStub;
+  let confirmStub: sinon.SinonStub;
+  let writeFileSyncStub: sinon.SinonStub;
+  let existsSyncStub: sinon.SinonStub;
+  let loggerInfoStub: sinon.SinonStub;
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox();
+    sandbox.stub(auth, "requireAuth").resolves();
+    needProjectIdStub = sandbox.stub(projectUtils, "needProjectId");
+    getAppConfigStub = sandbox.stub(apps, "getAppConfig");
+    getAppConfigFileStub = sandbox.stub(apps, "getAppConfigFile");
+    listFirebaseAppsStub = sandbox.stub(apps, "listFirebaseApps");
+    getOrPromptProjectStub = sandbox.stub(projects, "getOrPromptProject");
+    selectStub = sandbox.stub(prompt, "select");
+    confirmStub = sandbox.stub(prompt, "confirm");
+    writeFileSyncStub = sandbox.stub(fs, "writeFileSync");
+    existsSyncStub = sandbox.stub(fs, "existsSync");
+    loggerInfoStub = sandbox.stub(logger, "info");
+  });
+
+  afterEach(() => {
+    sandbox.restore();
+  });
+
+  it("should be a Command", () => {
+    expect(command).to.be.an.instanceOf(Command);
+  });
+
+  describe("action", () => {
+    it("should get config for a specified app", async () => {
+      getAppConfigStub.resolves({} as any);
+      getAppConfigFileStub.returns({ fileContents: "test-config" } as any);
+
+      await command.runner()("IOS", "test-app-id", {});
+
+      expect(getAppConfigStub).to.have.been.calledOnceWith("test-app-id", AppPlatform.IOS);
+      expect(loggerInfoStub).to.have.been.calledWith("test-config");
+    });
+
+    it("should get config for the only app when no app id is provided", async () => {
+      needProjectIdStub.returns("test-project-id");
```

Comment:
This one line appears in most of the tests. Should it be moved into the `beforeEach` block?